L1Trigger/L1TGlobal/src/GlobalBoard.cc

```diff
@@ -310,10 +316,33 @@ void l1t::GlobalBoard::receiveCaloObjectData(const edm::Event& iEvent,
 		  }
 */
 
-        }  //end loop over jet in bx
+        }  //end loop over EtSum objects in bx
       }    //end loop over Bx
     }
   }
+
+  if (receiveZdcEtSums) {
+    edm::Handle<BXVector<l1t::EtSum>> etSumData;
+    iEvent.getByToken(zdcEtSumInputToken, etSumData);
+
+    if (!etSumData.isValid()) {
+      if (m_verbosity) {
+        edm::LogWarning("L1TGlobal") << "\nWarning: BXVector<l1t::etSum> with input tag "
+                                     //<< caloInputTag
```

Comment:
Remove the comment ?
You actually removed a similar comment at lines `159` and `262`, it's weird that here it is being added.

On the other hand, could you make the warning a clearer that the missing collection is the ZDC energy sum, instead of the more standard one ?

Reply:
Right. All these comments do not make sense actually given that "which input tag" is commented in the cout statement so would be empty. I am changing all comments and making explicit the comment for Zdc energy sum (given that the dataformat is shared with standard EtSum so the message would be ambiguous).

Reply:
@elfontan fixes for above should be in the latest commit